framework/src/interfaces/BlockRestrictable.C

```diff
@@ -338,8 +338,9 @@ BlockRestrictable::checkVariable(const MooseVariableFieldBase & variable) const
 {
   // a variable defined on all internal sides does not need this check because
   // it can be coupled with other variables in DG kernels
-  if (variable.activeSubdomains().count(Moose::INTERNAL_SIDE_LOWERD_ID) > 0)
-    return;
+  for (const auto & id : variable.activeSubdomains())
+    if (_blk_mesh->interiorLowerDBlocks().count(id) > 0)
```

Comment:
this isnt quite the same check now. it's on ANY lowerD blocks when it used to be on ALL (just 1)

Reply:
so lets update the comment

Reply:
Good catch, I changed the code here to make the comment valid.